src/wallet/wallet.cpp

```diff
@@ -282,7 +282,13 @@ std::shared_ptr<CWallet> LoadWalletInternal(WalletContext& context, const std::s
         context.chain->initMessage(_("Loading wallet…"));
         std::shared_ptr<CWallet> wallet = CWallet::Create(context, name, std::move(database), options.create_flags, error, warnings);
         if (!wallet) {
-            error = Untranslated("Wallet loading failed.") + Untranslated(" ") + error;
+            bilingual_str output;
+            if (!error.empty()) output = error;
+            if (!warnings.empty()) {
+                if (!output.empty()) output += Untranslated("\n\n");
+                output = util::Join(warnings, Untranslated("\n"));
```

Comment:
In 86f05d81a12851ab68c33c2447a000cce52ee25c "wallet: init, don't error out when loading legacy wallets"

`output` is overwritten here, given the previous line, I assume it's supposed to be appended to?

Reply:
upps, yeah. Fixed.